guide/C/ch_txns.xml

```diff
@@ -349,10 +349,10 @@
         <note>
           <para>Note: An account register displays a transaction when that
           transaction has a split assigned to the account. If a given transaction
-          has more than one split assigned to a single account, then in Basic and
-          Auto-Split view modes that transaction will appear in the register one
+          has more than one split assigned to the register account, then in Basic and
```

Comment:
Well, I guess that's one way to read it. I didn't get that. I guess your new text works, though.